From 1e1fa1e260ddddb22c562cc9e84d6e48d7cd619c Mon Sep 17 00:00:00 2001
From: Jurgen Kramer <gtmkramer@xs4all.nl>
Date: Fri, 8 Dec 2017 09:09:25 +0100
Subject: [PATCH] ALSA: USB: Native DSD support for Armature Hectate

This patch adds native DSD support for the 'ARMATURE Hecate XMOS Xcore 208 USB SPDIF Asynchronous Interface'.

Signed-off-by: Jurgen Kramer <gtmkramer@xs4all.nl>
---
 sound/usb/quirks.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index a66ef57..6e6df57 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -1358,6 +1358,7 @@ u64 snd_usb_interface_dsd_format_quirks(struct snd_usb_audio *chip,
 	case USB_ID(0x20b1, 0x2009): /* DIYINHK DSD DXD 384kHz USB to I2S/DSD */
 	case USB_ID(0x20b1, 0x2023): /* JLsounds I2SoverUSB */
 	case USB_ID(0x20b1, 0x3023): /* Aune X1S 32BIT/384 DSD DAC */
+	case USB_ID(0x20b1, 0x3086): /* ARMATURE Hecate */
 	case USB_ID(0x2616, 0x0106): /* PS Audio NuWave DAC */
 		if (fp->altsetting == 3)
 			return SNDRV_PCM_FMTBIT_DSD_U32_BE;
-- 
2.9.5

